File: fiap/learningInPython/desviosCondicionais/desvioCompostoSeEntaoSenao.py
# venda = float(input("Digite a venda: "))
# if venda > 300 :
#   desconto = venda * 10 / 100
#  venda = venda - desconto
#  print("Novo valor: ", venda)
# ---

tc = int(input("Digite o seu tempo de casa"))
sal = float(input("Digite o valor do salario"))

if tc < 3:
    aumento = sal * 0.05
else:
    aumento = sal * 0.1
    novo_sal = sal + aumento
print("O seu salário foi de", sal, " para", novo_sal)

File: fiap/learningInJava/learningInJava/src/desviosCondicionais/desvioCompostoSeEntaoSenao.java
package desviosCondicionais;

import java.util.Scanner;

public class desvioCompostoSeEntaoSenao {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int tc;
        double sal,novo_sal,aumento;

        System.out.print("Digite o tempo de casa ");
        tc = input.nextInt();
        System.out.print("Digite o seu salario ");
        sal = input.nextDouble();
        if (tc < 3){
            aumento = sal * 0.05;
        }
else{
    aumento = sal * 0.1;
        }
novo_sal = sal + aumento;

System.out.print("O seu salario foi de " + sal + " para " + novo_sal);
    }
}
